src/modules/movie-list.js

```diff
@@ -23,11 +32,29 @@ const createList = async () => {
     title.textContent = `${content.name}`;
     movieCard.appendChild(title);
 
-    const like = document.createElement('img');
-    like.alt = 'Like button';
-    like.className = 'like-btn';
-    like.src = likeImage;
-    movieCard.appendChild(like);
+    const likeBtn = document.createElement('img');
+    likeBtn.alt = 'Like button';
+    likeBtn.className = 'like-btn';
```

Comment:
[OPTIONAL] I would suggest you add 'cursor: pointer' as a style to the like image. This would improve the website UI